website/versioned_docs/version-0.11.0/hoodie_deltastreamer.md

```diff
@@ -48,8 +48,15 @@ Options:
       Compaction is enabled for MoR table by default. This flag disables it
       Default: false
     --enable-hive-sync
-      Enable syncing to hive
+      Enable syncing to hive (Deprecated in favor of --enable-sync and --sync-tool-classes)
       Default: false
+    --enable-sync
+      Enable syncing meta
+      Default: false
+    --sync-tool-classes
+      Classes (comma-separated) to be used for syncing meta. Shall be used only when --enable-sync or --enable-hive-sync is set to true
+      Note: When used this with deprecated --enable-hive-sync flag, HiveSyncTool will always be run along with any other classes mentioned in here.
```

Comment:
nit: remove this

Reply:
Did you mean to remove the `Note:` section?

Reply:
Did you mean to remove the `Note:` section?

Reply:
Removed the note. Please check.

Reply:
No I meant to indicate this change - 

When used this with -> When used with

Reply:
Done

Reply:
Done